playbooks/roles/ambari-agent/tasks/main.yml

```diff
@@ -28,6 +28,11 @@
         (ansible_python_version is version_compare('2.7', '>=') and ((ansible_os_family|lower == "redhat" and ansible_distribution_major_version|int >= 7) or
                                                                      (ansible_os_family|lower == "suse" and ansible_distribution_major_version|int >= 12) or
                                                                      (ansible_os_family|lower == "redhat" and ansible_distribution|lower == "amazon")))
+- name: Comment out package_version = None in script.py
```

Comment:
What is this change for?

Reply:
I had issues with certain components failing to install. Error message was:
raise Fail("Cannot match package for regexp name {0}. Available packages: {1}".format(name, self.available_packages_in_repos)) 

Hortonworks Support pointed me to the thread below, which fixed the issue.
https://community.hortonworks.com/questions/203492/new-install-hdp-2650-292-via-ambari-2620-155-zooke.html